src/ouroboros/config.py

```diff
@@ -114,7 +114,8 @@ def create(category, name, *args, **kwargs):
         """Instantiate a specific config."""
         instance = ConfigFactory()
         if category not in instance._factories:
-            raise ValueError(f"No configs registered under category '{category}'")
+            print(f"WARNING: No configs registered under category '{category}'")
```

Comment:
I dropped this when I moved the config over, but I had integration with `logging` that was relatively nice that I meant to re-add (which I can do after the fact if you want)